packages/ohm-js/src/ohm-grammar.ohm

```diff
@@ -61,6 +70,12 @@ Ohm {
     | terminal                                       -- terminal
     | "(" Alt ")"                                    -- paren
 
+  include
+  	= caseInsensitive<"include">
```

Comment:
I don't think this should be case insensitive; it's not very common for languages to have case-insensitive keywords.